.github/workflows/fast.yml

```diff
@@ -150,6 +150,8 @@ jobs:
       run: rustup target add thumbv7em-none-eabihf
     - name: Build and test codec
       run: cargo build -p commonware-codec --no-default-features --target thumbv7em-none-eabihf --release && du -h target/thumbv7em-none-eabihf/release/libcommonware_codec.rlib && cargo test -p commonware-codec --no-default-features --verbose
+    - name: Build and test utils
+      run: cargo build -p commonware-utils --no-default-features --target thumbv7em-none-eabihf --release && du -h target/thumbv7em-none-eabihf/release/libcommonware_utils.rlib
```

Comment:
I didn't include the "cargo test" here because several of the tests currently rely on std features like format!